tests/integration/devfile/cmd_devfile_registry_test.go

```diff
@@ -0,0 +1,91 @@
+package devfile
```

Comment:
I think we also need to test cases where you run `odo create` from devfiles hosted in devfile registries that are added using the `registry` command. And similarly, that you can't `odo create` a component from a devfile registry that's been removed.